sdk/src/main/java/io/radar/sdk/RadarApiHelper.kt

```diff
@@ -114,6 +114,7 @@ internal open class RadarApiHelper(
                 urlConnection.disconnect()
             } catch (e: IOException) {
                 handler.post {
+                    logger?.e("Received network error | error = $e");
```

Comment:
Might be better to use `Throwable.toStackTraceString(e)`. We'll want to be cautious though - generally we won't want to _actually_ print stack traces. Instead, this should just get sent to the backend via the log endpoint and skip printing to console.